src/coingecko/Coingecko.ts

```diff
@@ -117,7 +117,7 @@ export class Coingecko {
         const { host } = this;
         const url = `${host}/${path}`;
         const result = await axios(url, { params: { x_cg_pro_api_key: this.apiKey } });
-        this.logger.debug({ at: "sdk-v2/coingecko", message: `Sent GET request to url ${result.request.responseURL}` });
+        this.logger.debug({ at: "sdk-v2/coingecko", message: `Sent GET request to url ${url}` });
```

Comment:
Ah - I have a [PR](https://github.com/across-protocol/sdk-v2/pull/79) up for this but hadn't realised you'd already proposed to fix it here.

I think it would be nice to migrate the message above the request submission. As-is, if the HTTP request fails then you never see the log message, and that's probably exactly when you want to see it. So I'd propose "Sent" -> "Sending" and move it up 1 line.